docs/self-managed/identity/orchestration-identity/installation.md

```diff
@@ -0,0 +1,105 @@
+
+Identity for Orchestration clusters is available via [Helm install](/self-managed/setup/install.md), and for local development via [Camunda 8 Run](/self-managed/setup/deploy/local/c8run.md).
+
+## Initial configuration
+
+Following the default installation for [Camunda 8 Run](/self-managed/setup/deploy/local/c8run.md) will result in a cluster with the following:
+
+1. Basic authentication enabled
+2. API authentication disabled
+3. Authorizations disabled
+4. An initial user with the username/password `demo`/`demo`
+5. An `admin` role with read, create, update, and delete permissions for other roles, which is applied to the initial `demo` user
+
+To make changes to the [initial configuration](./configuration.md), add the desired values to your `application.yaml`, and provide this file to Camunda 8 Run using the `--config` flag [at startup](/self-managed/setup/deploy/local/c8run.md#enable-authentication-and-authorization).
+
+:::note
+Helm installations have API authentication and authorizations enabled by default. To adjust your Helm configuration, add the desired values to your `application.yaml` or via environment variables.
+:::
+
+### Configure an initial user
+
+The initial user created by the application will be assigned the `admin` role, and can be used for authentication to the web applications and additional role management.
+
+To create a unique initial user, the following is required in your `application.yaml` or environment variables:
+
+<Tabs>
+  <TabItem value="helm" label="Helm properties" default>
+
+```yaml
+camunda:
+  security:
+    initialization:
+      users:
+        - username: <Your chosen username>
+          password: <Your chosen password>
+          name: <The name of the first user>
+          email: <The email address of the first user>
+```
+
+  </TabItem>
+<TabItem value="env" label="Environment variables" default>
+
+```shell
+CAMUNDA_SECURITY_INITIALIZATION_USERS[0]_USERNAME=<Your chosen username>
+CAMUNDA_SECURITY_INITIALIZATION_USERS[0]_PASSWORD=<Your chosen password>
+CAMUNDA_SECURITY_INITIALIZATION_USERS[0]_NAME=<The name of the first user>
+CAMUNDA_SECURITY_INITIALIZATION_USERS[0]_EMAIL=<The email address of the first user>
+```
+
+:::note
+Any other users included in the initialization `user` list will also be granted the `admin` role.
+:::
+
+  </TabItem>
+</Tabs>
+
+### Enable API authentication and authorizations
+
+In Camunda 8 Run installations, basic authentication is enabled on the Camunda web applications, but the API is unprotected, and authorizations are disabled. API protection and authorizations can both be enabled by modifying your `application.yaml` or environment variables:
+
+<Tabs>
+  <TabItem value="helm" label="Helm properties" default>
```

Comment:
🔧 I think this should have the `default` removed so the env variables tab is shown